include/pybind11/pybind11.h

```diff
@@ -114,14 +114,16 @@ class cpp_function : public function {
     object name() const { return attr("__name__"); }
 
 protected:
-    using unique_function_record = std::unique_ptr<detail::function_record, void (*)(detail::function_record *)>;
+    struct InitializingFunctionRecordDeleter {
```

Comment:
Just one more thing. Is this the only BigLetterCaseThing in the entire codebase?

Reply:
I'm not sure how to search for that. Do you prefer `snake_letter_case_thing`?